common/persistence/sql/storage/postgres/event.go

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2017 Uber Technologies, Inc.
```

Comment:
I believe events V1 is deprecated. plz remove this file and panic on usage of events v1 & postgres

Reply:
I removed both `event.go` files, but where can I deprecate the v1 api?

Reply:
we have 2 kinds of events table implementation, v1 and v2
v1 is deprecated since the beginning of this year.
v2 is the default.

v1 will not be used for prod purpose, kept it in the source code for long running workflows. perhaps we should do an analysis whether events v1 source code can be totally removed.

for your implementation, plz keep the interface as is, not the concrete implementation (just panic will do the work)